tests/io/v0_0_0/test_reader.py

```diff
@@ -24,17 +24,19 @@ def test_read_imagepartition(self):
     def test_read_tocpartition(self):
         result = slicedimage.Reader.parse_doc("tocpartition_l1.json", baseurl)
         self.assertIsInstance(result, slicedimage.TocPartition)
-        tocs = [toc for toc in result.all_tocs()]
+        tocs = [items for items in result.all_image_partitions()]
         self.assertEqual(len(tocs), 1)
-        self.assertEqual(tocs[0].shape, {'hyb': 4, 'ch': 4})
+        self.assertEqual(tocs[0][0], "fov_001")
+        self.assertEqual(tocs[0][1].shape, {'hyb': 4, 'ch': 4})
         self._verify_tiles(result.tiles())
 
     def test_read_multilevel_tpartition(self):
         result = slicedimage.Reader.parse_doc("tocpartition_l2.json", baseurl)
         self.assertIsInstance(result, slicedimage.TocPartition)
-        tocs = [toc for toc in result.all_tocs()]
+        tocs = [items for items in result.all_image_partitions()]
```

Comment:
I'd sort this by key, just to be safe.

Reply:
I don't understand this comment.

If anything, I would transform this to look up the toc by name.

Reply:
In fact, I'll do that.